tests/page.js

```diff
@@ -37,15 +37,18 @@ module.exports = function Page(browser) {
 	// elements which *might* be present in the DOM, hence the try/get name.
 
 	this.tryGetMainSectionElement = function () {
-		return this.tryFindByXpath('//section[@id="main"]');
+		return this.tryFindByXpath(
+			idSelectors ? '//section[@id="main"]' : '//section[contains(@class, "main")]');
```

Comment:
Any chance we could turn this into a function instead? Seems like an easy source for errors like this.